test/microphysics2M_tests.jl

```diff
@@ -142,10 +151,10 @@ function test_microphysics2M(FT)
         N_rai = [FT(1e1), FT(1e3), FT(1e5)]
         ρ = FT(1)
 
-        xr_min = FT(2.6e-10)
+        xr_min = FT(6.54e-11)
```

Comment:
```suggestion
     xr_min = SB2006.pdf_r.xr_min   
```

I think also something like this should work

`(; xr_min, xr_max, λ_min, λ_max) = SB2006.pdf_r`